interface/forms/bronchitis/table.sql

```diff
@@ -2,90 +2,90 @@ CREATE TABLE IF NOT EXISTS `form_bronchitis` (
 id bigint(20) NOT NULL auto_increment,
 date datetime default NULL,
 pid bigint(20) default NULL,
-user varchar(255) default NULL,
-groupname varchar(255) default NULL,
+user text default NULL,
+groupname text default NULL,
```

Comment:
should be just text (remove the default NULL)

Reply:
if concerned this may break something, then could just leave these 2 as is or make smaller varchar(50)